rollup.config.js

```diff
@@ -95,4 +95,11 @@ export default [
   createCommonJSConfig('src/shallow.ts', 'dist/shallow.js'),
   createCommonJSConfig('src/middleware.ts', 'dist/middleware.js'),
   createCommonJSConfig('src/vanilla.ts', 'dist/vanilla.js'),
+
+  // Create a purely ESM version, can be imported from `zustand/esm`
+  createESMConfig('src/vanilla.ts', 'dist/esm/vanilla.js'),
+  createESMConfig('src/shallow.ts', 'dist/esm/shallow.js'),
+  createESMConfig('src/index.ts', 'dist/esm/index.js'),
+  // I dont get the one below to succeed with rollup :(
+  // createModernESMConfig('src/middleware.ts', 'dist/esm/middleware.js'),
```

Comment:
😕

Reply:
Curious what wasn't working? This not only works out of the box for me, but also gets rid of a `regenerator-runtime` dependency which is great. Huge improvement for us.